addOns/domxss/src/main/java/org/zaproxy/zap/extension/domxss/DomXssScanRule.java

```diff
@@ -669,22 +669,18 @@ public boolean scan(String attackVector, String currUrl) {
 
         DomAlertInfo result = scanHelper(attackVector, currUrl);
         if (result != null) {
-            String tagName = result.getTagName();
-            String otherInfo = "";
-            if (tagName != null) {
-                otherInfo =
-                        "Tag name: "
-                                + tagName
-                                + " Att name: "
-                                + result.getAttributeName()
-                                + " Att id: "
-                                + result.getAttributeId();
+            StringBuilder otherInfo = new StringBuilder();
```

Comment:
It was not but I guess it should be i18n'ed?

Reply:
Ya that's probably best.